pio-scripts/pos-esp32.py

```
@@ -0,0 +1,301 @@
+        print()
+    for file in files:
+        if "no_files" in file:
+            print()
+            print(Fore.GREEN + "No files added -> will NOT create littlefs.bin and NOT overwrite fs partition!")
+            print()
+            continue
+        if "://" in file:
+            parts = file.split()
+            url = parts[0]
+
+            response = requests.get(url)
```

Ruff (0.14.10)

Probable use of `requests` call without timeout

(S113)